src/node_quic_session.cc

```diff
@@ -50,7 +50,7 @@ namespace quic {
 // QuicSession is an abstract base class that defines the code used by both
 // server and client sessions.
 QuicSession::QuicSession(
-    QuicSessionType type,
+    ngtcp2_crypto_side side,
```

Comment:
`ngtcp2_crypto_side` is actually defined in the newer version of ngtcp2 that we will be moving to. I'e gone ahead and adopted it here to make moving to the updated ngtcp2 in the next iteration easier.